verification.py

```diff
@@ -1269,30 +1263,43 @@ def _msms_params_to_run_msp(self, params):
                     position=params["sel_pos"],
                     end_frequency=params["end_frequency_lst"][i],
                     start_frequency=0.5 / params["refsize"],
-                    alpha=params["alpha"],
+                    s=params["alpha"] / params["refsize"],  # alpha=saA, s=sAA/(2N)
                     dt=1.0 / (40 * params["refsize"]),
                 )
                 t_start = params["end_time_lst"][i]
                 model.append((t_start, temp_model))
-            model.append((None, None))
+            model.append((None, "hudson"))
 
         else:
             model = "hudson"
 
-        scale_factor = params["num_sites"]
-        recombination_rate = params["rho"] / (scale_factor - 1)
+        scale_factor = 4.0 * params["refsize"] * (params["num_sites"] - 1)
+        recombination_rate = params["rho"] / scale_factor
+        scale_factor = 4.0 * params["refsize"] * params["num_sites"]
         mutation_rate = params["theta"] / scale_factor
 
-        ts = msprime.simulate(
-            sample_size=params["nsam"],
-            Ne=0.25,
-            length=params["num_sites"],
-            mutation_rate=mutation_rate,
-            recombination_rate=recombination_rate,
+        repeats = msprime.sim_ancestry(
+            samples=params["nsam"],
+            population_size=params["refsize"],
```

Comment:
Ah - these are diploid sample **individuals** now, we need ``params["nsam"] / 2`` here.

If this number is odd, then you can do
```
samples=msprime.SampleSet(params["nsam"], ploidy=1)
```
to achieve the same effect (i.e., keeping the timescale diploid but sampling haploid individuals)

Reply:
yeah this will probably account for a lot of the differences in these tests @gbinux

Reply:
Good catch! I tried `params["nsam"] / 2`  and avoided using odd numbers. It seems the discrepancies are still there. 
[MsmsSweeps_3.zip](https://github.com/tskit-dev/msprime/files/6190809/MsmsSweeps_3.zip)

Before I keep pushing the code, I want to show my understanding of the relevant parameters/options. Maybe listing them out could help find the issue if it is obvious. The following linked docs is also a draft related to   #1286 

[Compare args/options between different coalescent simulators](https://docs.google.com/document/d/1TmkfWN5hREOj0wy_B6K3OALAYIydgdPTaulImKheQ18/edit?usp=sharing)

Are there additional misunderstandings about the parameters? Is this kind of information useful to be included in the docs? If the discrepancies persist after we excluded the parameter scaling issue, are we going to keep this Test class within the msprime 1.0 release? @jeromekelleher @andrewkern

Reply:
Hmm, these still look like pretty big differences to me. Your scalings look about right to me in the doc, but I'm not that well up on these other programs any more.

Can you summarise what you think the differences are, and where we're seeing deviations between the programs please?

Reply:
Hi @gbinux-- we already know that discoal, msprime, and slim give the same results from my additions to `verification.py`. you might just look in the `DiscoalSweep` and `SweepVsSlim` test classes in there to see where your parameterization conversion differs.